modules/bibauthorid/lib/bibauthorid_hoover.py

```diff
@@ -0,0 +1,746 @@
+    write_message("Initializing hoover", verbose=1)
+    write_message("Selecting records with identifiers...", verbose=1)
+    recs = get_records_with_tag('100__i')
+    task_sleep_now_if_required(can_stop_too=True)
+    recs += get_records_with_tag('100__j')
+    task_sleep_now_if_required(can_stop_too=True)
+    recs += get_records_with_tag('700__i')
+    task_sleep_now_if_required(can_stop_too=True)
+    recs += get_records_with_tag('700__j')
+    task_sleep_now_if_required(can_stop_too=True)
+    write_message("Found {0} records".format(len(set(recs))), verbose=2)
+    recs = set(recs) & set(
+        run_sql("select DISTINCT(bibrec) from aidPERSONIDPAPERS"))
+    write_message("   out of which {0} are in BibAuthorID".format(len(recs)), verbose=2)
+    task_sleep_now_if_required(can_stop_too=True)
+
+    records_with_id = set(rec[0] for rec in recs)
+
+    destroy_partial_marc_caches()
+    populate_partial_marc_caches(records_with_id, create_inverted_dicts=True)
+
+    if rt_ticket_report:
+        global ticket_hashes
+        write_message("Ticketing system rt is used", verbose=9)
+        write_message("Building hash cache for tickets for queue %s" % queue, verbose=9)
+        ticket_ids = BIBCATALOG_SYSTEM.ticket_search(None, subject='[Hoover]', queue=queue)
+        write_message("Found %s existing tickets" % len(ticket_ids), verbose=9)
+        for ticket_id in ticket_ids:
+            task_sleep_now_if_required(can_stop_too=True)
+            try:
+                ticket_data = BIBCATALOG_SYSTEM.ticket_get_info(
+                    None, ticket_id)
+                ticket_hashes[
+                    ticket_data['subject'].split()[-1]] = ticket_data, ticket_id, False
+            except IndexError:
+                write_message("Problem in subject of ticket {0}".format(ticket_id), verbose=5)
+        write_message("Found {0} tickets".format(len(ticket_hashes)), verbose=2)
+
+    task_sleep_now_if_required(can_stop_too=True)
+    fdict_id_getters = {
+        "INSPIREID": {
+            'reliable': [get_inspire_id_of_author,
+                         get_inspireID_from_hepnames,
+                         lambda pid: get_inspireID_from_claimed_papers(
+                             pid, intersection_set=records_with_id, queue=queue)],
+
+            'unreliable': [lambda pid: get_inspireID_from_unclaimed_papers(
+                           pid, intersection_set=records_with_id, queue=queue)],
+            'signatures_getter': get_signatures_with_inspireID,
+            'connection': dict_entry_for_hepnames_connector,
+            'data_dicts': {
+                'pid_mapping': defaultdict(set),
+                'id_mapping': defaultdict(set)
+            }
+        },
+
+        "ORCID": {
+            'reliable': [  # get_orcid_id_of_author,
+                # get_inspireID_from_hepnames,
+                # lambda pid: get_inspireID_from_claimed_papers(pid,
+                # intersection_set=records_with_id)]
+            ],
```

Comment:
Why orcid is commented out :cry: . We should totally enable it. What is missing?